Source/VOE/VOE.csproj

```
@@ -76,7 +73,9 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="packages.config" />
+    <PackageReference Include="Lib.Harmony">
+      <Version>2.2.1</Version>
+    </PackageReference>
```

```suggestion
    <PackageReference Include="Lib.Harmony">
      <Version>2.2.1</Version>
      <ExcludeAssets>runtime;</ExcludeAssets>
    </PackageReference>
```
Just ensures we don't get a `0Harmony` dll in the mod by accident.